frontend/src/actions/followers.js

```diff
@@ -101,9 +101,9 @@ export const checkFollower = (foreignAuthorId) => (dispatch, getState) => {
   axios
     .get(
       `/author/${authorId}/followers/${foreignAuthorId}`,
-      tokenConfig(getState)
-    )
-    .then((res) => {
+      tokenConfig(getState))
+      .then((res) => {
+      console.log("response - ", res);
```

Comment:
why is token config being redone here? you should import it from auth 
also, remove this console log